editor/src/components/navigator/navigator-utils.ts

```diff
@@ -132,7 +133,12 @@ export function getNavigatorTargets(
       ) {
         if (elementFromProjectContents != null) {
           // add synthetic entry
-          const dataRefEntry = dataReferenceNavigatorEntry(path, elementFromProjectContents)
+          const dataRefEntry = dataReferenceNavigatorEntry(
+            path,
+            EP.parentPath(path),
+            elementFromProjectContents,
+            PP.create('children'),
```

Comment:
hmmmm I don't like this, 
1. it doesn't currently work
2. I think this will likely never work because the children prop can receive more than a single JSX element as child, see for example:
```
      <Heading>Hello {copy}</Heading>
```

where we already render 'Hello ' and {copy} as two sibling cartouches in the navigator!


Instead of `renderedByElementPath` and `propertyPath`, it'd be neater if `DataReferenceNavigatorEntry` would take a singe, _optional_ ElementPropertyPath:

```typescript
export interface DataReferenceNavigatorEntry {
  type: 'DATA_REFERENCE'
  elementPath: ElementPath
  childOrAttribute: JSXElementChild
  renderedAt: ElementPropertyPath | null
}
```

and here for this case, provide a null for it

Reply:
implemented it in https://github.com/concrete-utopia/utopia/pull/5609/commits/097458c020c6a602d59417f0db64026d6fe40e42